### PR TITLE
Added task 2079

### PR DESCRIPTION
**Pull request by AryamanMishra**

### CODE REVIEW

File: src/main/java/g2001_2100/s2079_watering_plants/readme.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+You have to refill before watering each plant. 
+
+Steps needed = 1 + 1 + 2 + 2 + 3 + 3 + 4 + 4 + 5 + 5 + 6 + 6 + 7 = 49.
+
+**Constraints:**
+
+* `n == plants.length`
+* `1 <= n <= 1000`
+* `1 <= plants[i] <= 10<sup>6</sup>`
```
Comment (javadev) [inline src/main/java/g2001_2100/s2079_watering_plants/readme.md]: It looks weird. Please use generatemd tool.

File: src/main/java/g2001_2100/s2079_watering_plants/readme.md [hunk 2]
```diff
@@ -62,13 +62,13 @@ Steps needed = 3 + 3 + 4 + 4 + 5 + 5 + 6 = 30.
...
 
 Steps needed = 1 + 1 + 2 + 2 + 3 + 3 + 4 + 4 + 5 + 5 + 6 + 6 + 7 = 49.
 
 **Constraints:**
 
 * `n == plants.length`
 * `1 <= n <= 1000`
-* `1 <= plants[i] <= 10<sup>6</sup>`
-* `max(plants[i]) <= capacity <= 10<sup>9</sup>`
+* `1 <= plants[i] <= 10⁶`
```
Comment (javadev) [inline src/main/java/g2001_2100/s2079_watering_plants/readme.md]: It should be code tag.

Generatemd tool https://github.com/javadev/LeetCode-in-Java/releases/download/generatemd/generatemd-1.0.jar

File: src/main/java/g2001_2100/s2079_watering_plants/readme.md [hunk 3]
```diff
@@ -69,6 +69,6 @@ Steps needed = 1 + 1 + 2 + 2 + 3 + 3 + 4 + 4 + 5 + 5 + 6 + 6 + 7 = 49.
 **Constraints:**
 
 * `n == plants.length`
-* `1 <= n <= 1000`
-* `1 <= plants[i] <= 10⁶`
-* `max(plants[i]) <= capacity <= 10⁹`
+* `<code>`1 <= n <= 1000 `</code>`
+* `<code>`1 <= plants[i] <= 10 `<sup>`6 `</sup></code>`
```
Comment (javadev) [inline src/main/java/g2001_2100/s2079_watering_plants/readme.md]: It looks mix tag and tilda. Please use only tags.

**Comment by AryamanMishra** [comment]
This is wrong too. I will do again

**Comment by javadev** [comment]
👍

**Pull request merged**